python/ray/tune/logger.py

```diff
@@ -103,17 +104,31 @@ def close(self):
         self.local_out.close()
 
 
+def to_tf_values(result, path):
+    values = []
+    for attr in result.keys():
+        value = result[attr]
+        if value is not None:
```

Comment:
less verbose to just do `for attr, value in result.items()`

Reply:
Done